projects/m1/026-is-a-string-a-palindrome/solutions/stefanocroci/sol_026_is_a_string_a_palindrome.py

```diff
@@ -0,0 +1,30 @@
+user = input('Insert a string to know if it is a palindrome: ')
+
+# Create a bool to save if not a palindrome
+is_a_palindrome = True
+for i in range(len(user)):
+    # If i is more than len() - 1 - i break the loop to avoid to repeat the same check again
+    if i > len(user) - (i + 1):
+        break
+    # If user[i] is different than len() - 1 - i then it's not a palindrome
+    if user[i].lower() != user[len(user) - (i + 1)].lower():
+        print(user + ' is not a palindrome')
+        is_a_palindrome = False
+        break
+
+if is_a_palindrome == True:
+    print(user + ' is a palindrome')
+
+
+# Method two with exit()
```

Comment:
Looks like this solution is more compact than the first. May I ask you why did you preferred the first one?

Reply:
I wanted to try using bools. it seemed to me a slightly less obvious solution